php_phongo.c

```diff
@@ -1742,22 +1856,22 @@ zend_object_iterator *php_phongo_cursor_get_iterator(zend_class_entry *ce, zval
 /* {{{ Memory allocation wrappers */
 static void* php_phongo_malloc(size_t num_bytes) /* {{{ */
 {
-	return emalloc(num_bytes);
+	return pemalloc(num_bytes, 1);
```

Comment:
I'm not sure either. I don't think we ought to be setting the libbson memory functions anymore.